### PR TITLE
Asset stats update

### PR DESCRIPTION
**Pull request by byterose**

### CODE REVIEW

File: src/components/ugas/GasStats.vue [hunk 1]
```diff
@@ -441,7 +441,7 @@ ul {
   align-items: baseline;
   font-size: 0.875rem;
   line-height: 1.25rem;
-  font-weight: 600;
+  font-weight: 600px;
```
Comment (ethedev) [inline src/components/ugas/GasStats.vue:444]: font weight is never in pixels
```suggestion
  font-weight: 600;
```

File: src/views/Asset.vue [hunk 2]
```diff
@@ -523,6 +526,8 @@ export default {
       navAct: "mint",
       info: true,
       tokenSelected: null,
+      showInfo: false,
+      showInfoButtonText: "More Info",
```
Comment (ethedev) [inline src/views/Asset.vue:530]: should keep same aspect unless something is on purpose
```suggestion
      showInfoButtonText: "Gas Info",
```

File: src/components/ugas/GasStats.vue (hunk 1 above)
Comment (byterose) [reply src/components/ugas/GasStats.vue]: I did this because this somehow fixed a safari bug - but have now undone it.

Comment (byterose) [reply src/components/ugas/GasStats.vue]: I did this because this fixed a safari bug - but have now undone it.

**Pull request merged**